Scripts/CreateTestingPR.ps1

```diff
@@ -34,7 +34,7 @@ if ($firstRun)
     & git reset --hard
     & git checkout -b $preBranch
 
-    dotnet $csharpierProject\Src\CSharpier\bin\Release\net5.0\dotnet-csharpier.dll
+    dotnet $csharpierProject\Src\CSharpier\bin\Release\net5.0\dotnet-csharpier.dll .
```

Comment:
probably worth making a `$csharpier` variable like you do in `TestCli.ps`.

Reply:
Also, github won't let me make comments on lines that are not part of the diff, but it's probably worth putting `Release` into a variable and referencing it here and in the build command.

You'll probably never change build configuration, but I'm a stickler for not repeating code :smirk: